From 93531bf4348fe3508ab95f8b1069c368b3051461 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Thu, 29 Aug 2019 15:55:20 +0300
Subject: [PATCH] configs: rcar-gen3-common: Add generic filesystem load
 command

This adds CONFIG_CMD_FS_GENERIC to rcar-gen3-common.h
which enables using generic filesystem "load" command.
The command should autodetect filesystem type and call
the appropriate load command wrapper.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 include/configs/rcar-gen3-common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/rcar-gen3-common.h b/include/configs/rcar-gen3-common.h
index c574ac3..5755aba 100644
--- a/include/configs/rcar-gen3-common.h
+++ b/include/configs/rcar-gen3-common.h
@@ -34,6 +34,7 @@
 #define CONFIG_CMD_CACHE
 #define CONFIG_CMD_GPIO
 #define CONFIG_CMD_LOADB
+#define CONFIG_CMD_FS_GENERIC
 
 #define CONFIG_CMD_SOURCE
 #define CONFIG_CMD_ECHO
-- 
2.7.4

